src/en/space-station-14/player-interaction/proposals/powered-hardsuits.md

```diff
@@ -0,0 +1,97 @@
+
+This exists to keep people from wearing them 24/7 as people did with modsuits, using less abilities would drain charge slower, allowing people to disable things when unneeded to conserve power, or just never use them for extended spacesuit time. The same applies to antagonists, though your blood-red hardsuit has a much higher energy capacity than any of the crew's hardsuits, you'll still have to charge it at some point, and when that time comes you'll be extremely vulnerable. No more syndies buying a jetpack and a blood-red and just flying around the station for the entire round shooting people.
+
+Running out of charge would not prevent the suit from protecting you from space, but it would give you a major movement speed decrease (think 25%-75% depending on the suit) and entirely prevent you from using ANY abilities. Stored items will be locked inside, magboots will be turned off, and additional armor shields will be disabled. Engineers could still wrap up repairs after running out of power if they have other equipment rather than relying on their suits, and security could finish dragging an apprehended suspect back to their department, just very slowly.
+
+## Suit Abilities
+
+Specific hardsuits would have specific abilities that could be seen in SS13 modsuits, but these abilities would be hard-locked to specific suits, rather than being placed on any suit with a module. The abilities could include...
+
+- **A basic flashlight on the helmet/suit.**
+A small light with a low energy cost that lets you see. You could keep it off in a lit area, or if you want to conserve suit charge. Carrying a seclight or a normal flashlight is also viable.
+- **Engi hardsuits giving welding protection.**
+Maybe you want to wear sunglasses or a medhud or just don't like how engi goggles look on your character, well it doesn't come free, since having this turned on would also cost suit charge. An efficient player could turn it on/off whenever they're welding, but that runs the risk of forgetting to turn it on and damaging your eyes.
+- **Engi hardsuits having built-in magboots.**
+Perhaps you didn't grab magboots before gravity went out or stole an engi suit and don't have access to them at all, either way, you still need to use suit charge to power them.
+- **Atmos hardsuits having thermal regulators that when disabled don't protect you from extreme heat.**
+A similar situation to the welding goggles ability. You can either leave it turned on at all times while wearing the suit, or you can turn it on/off while you're dealing with a recent plasma leak and subsequent fire. Conserving suit charge at slight risk of forgetting and causing your death.
+- **Mining/luxury hardsuits having built-in appraisal tools on item examination.**
```

Comment:
Exoskeleton ability to speed up carrying boxes short distances, have low, medium, and high settings for hardsuit lights maybe.